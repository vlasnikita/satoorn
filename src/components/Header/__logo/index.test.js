import React from "react";
import ShallowRenderer from "react-test-renderer/shallow";

import Logo from "./index";

it('should render Logo correctly', () => {
    const renderer = new ShallowRenderer();
    const wrapper = renderer.render(<Logo />)
    expect(wrapper).toMatchSnapshot()
})