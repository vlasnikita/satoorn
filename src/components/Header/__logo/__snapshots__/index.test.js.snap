// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Logo correctly 1`] = `
<img
  src="logo.svg"
/>
`;
